TWG/MeetingMinutes/2020-11-30-TWG-Minutes.md.md

```diff
@@ -0,0 +1,73 @@
+TWG Meeting Report
+Nov 30/2020
+
+## TG Reports
+
+### SW TG
+
+Written report prepared - thanks to Jeremy:
+https://github.com/openhwgroup/core-v-docs/blob/master/program/TG-reports-for-TWG/20201130-sw-tg.md
+
+IDE - Nearing CQ. Most important issue is "hello world application".
+
+GNU Tools - Good progress. Need to have FSF assignment from contributors - Action to Duncan. 

```

Comment:
Need to distinguish two actions

* OpenHW and any contributor having FSF copyright papers in place. This is a requirement for upstream contribution.  OpenHW having copyright papers in place means i) anything owned by OpenHW Group can be contributed upstream and ii) for contributors who have difficultly getting FSF copyright papers can be seconded to OpenHW Group and contribute as OpenHW Group staff.  Existing action on Duncan to get FSF copyright papers signed.

* Contributors having an agreement with OpenHW that anything they contribute to OpenHW Group repositories may subsequently be contributed upstream by OpenHW Group (giving credit to the originator).  Without this, contributions would be eligible to be upstreamed, but OpenHW Group would not have permission to contribute the work upstream.  Action on Duncan to create such an agreement for contributors to sign.

Reply:
I wasn't putting this level of notes on the TG reports, but have now included your text as you have spelled it out so clearly.